### PR TITLE
Alerting when a service cluster is smaller than it should be

### PR DESCRIPTION
**Pull request by stkem**
Not super elegant and it doesn't take into account a services status, but this should let us know when we loose an instance. cc @eishay @andrewconner

### CODE REVIEW

**Comment by eishay** [comment]
cool

File: kifi-backend/modules/common/app/com/keepit/common/service/Service.scala [hunk 1]
```diff
@@ -21,6 +21,10 @@ sealed abstract class ServiceType(val name: String, val shortName: String) {
   def selfCheck() : Future[Boolean] = promise[Boolean].success(true).future
   def healthyStatus(instance: AmazonInstanceInfo): ServiceStatus = ServiceStatus.UP
   override def toString(): String = name
+
+  val minInstances  : Int = 1
+  val warnInstances : Int = 2
```
Comment (raymondng) [inline kifi-backend/modules/common/app/com/keepit/common/service/Service.scala:26]: Thanks! The trade-off is that there's now one more place to do book keeping when we bring up/down instances ... it'd be nice if we can do this algorithmically (75%, 50%, etc.), or at least make this configurable via a management console (i.e. admin page).

Comment (stkem) [inline kifi-backend/modules/common/app/com/keepit/common/service/Service.scala:26]: Agreed. This is mostly a stop gap until we can come up with a better way, ideally not within a service.
Also, if we add machines you don't _need_ to update this and we probably won't ever remove machines.

Comment (raymondng) [inline kifi-backend/modules/common/app/com/keepit/common/service/Service.scala:26]: lol

**Comment by raymondng** [comment]
Nice. Thanks Stephen!

**Pull request merged**